tags/tags.toml

```
@@ -1158,3 +1158,10 @@ Rate limits are determined based on the IP address of the sender.
 • If your bot shares its IP with other bots it will share their rate limits.
 • This is a common caveat with [shared hosting](https://en.wikipedia.org/wiki/Shared_web_hosting_service) providers and free plans!
 """
+
+[application-command-permissions]
+keywords = ["application-command-permissions", "command-permissions", "permissions"]
```

```suggestion
keywords = ["application-command-permissions", "command-permissions", "slash-permissions"]
```

as permissions aren't accurate 
and I personally would type slash to get the tag